### ISSUE TITLE
Build failure on Carthage due to lack of formatting tools

### ISSUE THREAD

**Issue by bbqsrc** [dennisweissmann/DeviceKit#70]
```
swiftformat not found. Trying to install it via Homebrew...
Installing swiftformat via homebrew failed. Homebrew not found. Aborting.
```

External dependencies that format code or lint code should not be required for creating a release build.

As it stands, I can't build DeviceKit using Carthage.

**Comment by dennisweissmann** [comment]
Nice catch! I wanted to rework this for a while now (make it a pre-commit hook) but haven't found the time yet. Sorry for the trouble, as a workaround you can delete the build phase script inside the checked out repository ... sorry for the trouble!

**Comment by dennisweissmann** [comment]
Resolved by #71 

Thanks again!
